### ISSUE TITLE
Possibly automate version builds

### ISSUE THREAD

**Issue by niden** [phalcon/docker#6]
I am not sure if this is feasible.

For now, the dockerfiles contain the version of Phalcon in a variable

`ARG PHALCON_VERSION=5.0.4`

I am wondering if there is a way to change that and thus initiate a build whenever we release in the main repository.

**Comment by MilesChou** [comment]
It's possible for Automatic build In Docker Hub, it's can trigger by tag and pass the tag as build arg. But Automatic build is non-free.

At GitHub Action, maybe [this article](https://blog.marcnuri.com/triggering-github-actions-across-different-repositories) can help us. Use the GitHub API.

**Comment by Jeckerson** [comment]
Yes, it is possible to. Make matrix with Phalcon version dynamic.
